examples/k8s/otel-config.yaml

```diff
@@ -12,9 +12,9 @@ data:
       otlp:
         protocols:
           grpc:
-            endpoint: ${env:MY_POD_IP}:4317
+            endpoint: 0.0.0.0:4317
```

Comment:
We are trying to not do this, see https://github.com/open-telemetry/opentelemetry-collector/issues/6151